### PR TITLE
Add pycryptodome, python-pillow and simplejson

### PR DESCRIPTION
**Pull request by gipawu**
It is needed for Netflix plugin (https://github.com/asciidisco/plugin.video.netflix)

### CODE REVIEW

File: deps/pycryptodome.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "name": "pycryptodome",
+    "buildsystem" : "simple",
+    "build-commands" : [
+        "touch .separate_namespace",
+        "pip install . --prefix=/app"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://github.com/Legrandin/pycryptodome/archive/v3.7.3x/pycryptodome-3.7.3x.tar.gz",
```
Comment (Marko10-000) [inline deps/pycryptodome.json]: https://files.pythonhosted.org/packages/c7/ff/1ca71a40eb69c89778396a30d399639d41473b09c36aff2b700d80dd94b9/pycryptodome-3.7.3.tar.gz
sha256: 1a222250e43f3c659b4ebd5df3e11c2f112aab6aef58e38af55ef5678b9f0636

File: deps/python-pillow.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "python-pillow",
+    "buildsystem" : "simple",
+    "build-commands" : [
+        "pip install . --prefix=/app"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url" : "https://github.com/python-pillow/Pillow/archive/5.4.1/Pillow-5.4.1.tar.gz",
```
Comment (Marko10-000) [inline deps/python-pillow.json]: https://files.pythonhosted.org/packages/3c/7e/443be24431324bd34d22dd9d11cc845d995bcd3b500676bcf23142756975/Pillow-5.4.1.tar.gz
sha256: 5233664eadfa342c639b9b9977190d64ad7aca4edc51a966394d7e08e7f38a9f

File: deps/simplejson.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "simplejson",
+    "buildsystem" : "simple",
+    "build-commands" : [
+        "pip install . --prefix=/app"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url" : "https://github.com/simplejson/simplejson/releases/download/v3.16.1/simplejson-3.16.1.tar.gz",
```
Comment (Marko10-000) [inline deps/simplejson.json]: https://files.pythonhosted.org/packages/e3/24/c35fb1c1c315fc0fffe61ea00d3f88e85469004713dab488dee4f35b0aff/simplejson-3.16.0.tar.gz
sha256: b1f329139ba647a9548aa05fb95d046b4a677643070dc2afc05fa2e975d09ca5

**Comment by wsnipex** [comment]
thanks, please squash to 1 commit per new dependency

**Comment by achimfrase** [comment]
Any chance to get this updated soon?

**Comment by wsnipex** [comment]
@Marko10-000 OK for you?

**Pull request merged**